src/panels/mailbox/ha-panel-mailbox.js

```diff
@@ -212,10 +212,29 @@ class HaPanelMailbox extends LocalizeMixin(PolymerElement) {
     var platform = event.model.item.platform;
     this.currentMessage = event.model.item;
     this.$.mp3dialog.open();
-    this.$.mp3src.src = '/api/mailbox/media/' + platform + '/' + event.model.item.sha;
     this.$.transcribe.innerText = event.model.item.message;
-    this.$.mp3.load();
-    this.$.mp3.play();
+    // Pass authorization in request:
+    //   https://stackoverflow.com/questions/46878787/
+    //           how-to-set-loaded-audio-to-a-html-audio-tag-controller
+    var url = '/api/mailbox/media/' + platform + '/' + event.model.item.sha;
+    var oReq = new XMLHttpRequest();
+    var mp3 = this.$.mp3;
+    var auth = this.hass.connection.options;
```

Comment:
This is no longer correct since #1612. The frontend will now only use the new auth .

Reply:
Also why wouldn't you be able to use `hass.callApi` ?

Reply:
Because I need access the the response (not the responsebody)  in order to create URL.createObjectURL().  I don't know how to get access to that from hass.callApi(), and I don't know how to build a URL that the audio component will accept without it.  If there is a way, I would love to use it so I don't duplicate the authentication logic

Reply:
FYI, I did try to convert the responsetext back to a blob:
```
blob = new Blob([result], {type: 'audio/mpeg'});
```
But the audio component didn't like that

Reply:
Alternatively, I could handle this like the camera component does, and use a token, which would let me go back to just assigning the audio.src value.  Is that preferable?